e2e/cephfs.go

```diff
@@ -1213,6 +1213,142 @@ var _ = Describe("cephfs", func() {
 				}
 
 				validateSubvolumeCount(f, 0, fileSystemName, subvolumegroup)
+
+				err = deleteResource(cephFSExamplePath + "snapshotclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete CephFS snapshotclass: %v", err)
+				}
+			})
+
+			By("checking snapshot-backed volume", func() {
+				err := createCephfsStorageClass(f.ClientSet, f, false, nil)
+				if err != nil {
+					e2elog.Failf("failed to create CephFS storageclass: %v", err)
+				}
+
+				err = createCephFSSnapshotClass(f)
+				if err != nil {
+					e2elog.Failf("failed to delete CephFS storageclass: %v", err)
+				}
+
+				pvc, err := loadPVC(pvcPath)
+				if err != nil {
+					e2elog.Failf("failed to load PVC: %v", err)
+				}
+				pvc.Namespace = f.UniqueName
+				err = createPVCAndvalidatePV(f.ClientSet, pvc, deployTimeout)
+				if err != nil {
+					e2elog.Failf("failed to create PVC: %v", err)
+				}
+
+				app, err := loadApp(appPath)
+				if err != nil {
+					e2elog.Failf("failed to load application: %v", err)
+				}
+				app.Namespace = f.UniqueName
+				app.Spec.Volumes[0].PersistentVolumeClaim.ClaimName = pvc.Name
+				appLabels := map[string]string{
+					appKey: appLabel,
+				}
+				app.Labels = appLabels
+				optApp := metav1.ListOptions{
+					LabelSelector: fmt.Sprintf("%s=%s", appKey, appLabels[appKey]),
+				}
+				err = writeDataInPod(app, &optApp, f)
+				if err != nil {
+					e2elog.Failf("failed to write data: %v", err)
+				}
+
+				appTestFilePath := app.Spec.Containers[0].VolumeMounts[0].MountPath + "/test"
```

Comment:
good to define as a global var ? so that other app mounts can also use the same?

Reply:
The path is not consistent across all pod-related manifests in `examples/cephfs`. E.g.:

https://github.com/ceph/ceph-csi/blob/959df4dbac9c23f8b302b768855e2ca21ae60afe/examples/cephfs/pod.yaml#L10-L12

and

https://github.com/ceph/ceph-csi/blob/959df4dbac9c23f8b302b768855e2ca21ae60afe/examples/cephfs/pod-restore.yaml#L10-L12

Or do you mean just the `/test` filename bit?

Reply:
`Or do you mean just the /test filename bit?` -> true.. also I had noticed the mountPath varies in different tests, I was thinking to have a common mountPath for e2e tests where ever we exec and check or check on the app pod mount point.. but again, please feel free to discard this comment for now, but if we can unify it, that would be good as well! instead of keep having `path` different in different tests.